puppet/fabfile.py

```diff
@@ -106,8 +106,9 @@ def backup_db(dbname = 'rams_db', local_backup_dir='~/backup/'):
     sudo("chown postgres.postgres -R " + backups_dir)
     sudo("chmod 700 " + backups_dir)
 
-    backup_cmd = 'pg_dump ' + dbname + ' -f ' + remote_backup_fullpath
-    sudo("su - postgres -c '" + backup_cmd + "'")
+    run('db_name=$(facter db_name); echo "db_name=\'${db_name:-' + dbname + '}\'"')
```

Comment:
This line is just echoing the db_name to make sure we're grabbing the right thing.
```bash
db_name=$(facter db_name); echo "db_name='${db_name:-default_value}'"
```